ui.apps/src/main/content/jcr_root/apps/saas-aem-module/components/saas/search/search.html

```diff
@@ -0,0 +1,31 @@
+<sly data-sly-use.model="com.valtech.aem.saas.api.fulltextsearch.Search"></sly>
+<sly data-sly-use.template="/libs/core/wcm/components/commons/v1/templates.html"
+     data-sly-call="${template.placeholder @ isEmpty=wcmmode.edit}"></sly>
+<div class="saas-aem-module-search">
+  <form class="saas-aem-module-search__form"
+        id="searchForm"
+        action="${currentPage.path @ extension='html'}">
+    <input data-sly-test="${wcmmode.disabled}"
```

Comment:
- [ ] Why is it there? It should be added only in edit mode (if needed at all), but not in disabled mode that would be visible productive

Reply:
The markup is a draft version whose purpose is only to demo the current BE implementation.
I'd say we'll have the definite version when FE logic is implemented.

The above input field is of type hidden. It is there to add wcmmode=disabled on submit.